internal/target/koji_target.go

```diff
@@ -1,7 +1,7 @@
 package target
 
 type KojiTargetOptions struct {
-	Filename        string `json:"filename"`
+	Filename        string `json:"filename"` // Filename of the image as produced by osbuild for a given export
```

Comment:
Nitpick but moving the comment to the line above the field name would make it the docstring by convention and I think looks nicer.

Not an issue though :)
I think it gets picked up as the docstring like this as well.

Reply:
I'll change it, because in addition to that I noticed that the change in `v2_koji_test.go` from the second commit should have been in the first one.

Reply:
I fixed the nitpick and moved the test change to the first commit.... The PR should be perfect now 😇